doc/Output JSON.json

```diff
@@ -322,22 +351,22 @@
           "text": "Menstrual cycle"
         },
         "entry": [ {
-          "reference": "Observation/8973a466-6dc4-4c6c-8e04-aac141f3bddd"
+          "reference": "Observation/8da7eab5-c5c6-49d6-810f-aee45f2d80aa"
         } ]
       }, {
         "title": "Investigation Advice",
```

Comment:
Only one section for "Investigation Advice" that can have multiple entries. also use code SNOMED "721963009" for the section. Please refer to the ABDM profile